src/Data/Text/Lazy/Encoding.hs

```diff
@@ -161,6 +162,9 @@ encodeUtf8Builder :: Text -> B.Builder
 encodeUtf8Builder =
     foldrChunks (\c b -> TE.encodeUtf8Builder c `mappend` b) Data.Monoid.mempty
 
+encodeASCII :: Text -> B.ByteString
+encodeASCII = encodeUtf8
```

Comment:
this is a lie.

Reply:
Yes, this is just a placeholder, I didn't get to implementing the lazy version yet.

Reply:
I usually liberally leave `TODO`s in the code, as hint for myself and others, but also as a reminder "what else I had to do". It's then hopefully easier to see whether everything I had in mind is done when I self-review the patch.